psydac/linalg/basic.py

```diff
@@ -881,8 +881,8 @@ def transpose(self):
     def inverse(self, solver, **kwargs):
         return self._linop
 
-    @staticmethod
-    def jacobi(A, b):#, out=None):
+    #@staticmethod
+    def jacobi(A, b, out=None):
```

Comment:
If your remove the `@staticmethod` decorator you also need to pass `self` as the first function argument.